test/cppsuite/test_harness/util/scoped_types.h

```diff
@@ -26,76 +26,44 @@
  * OTHER DEALINGS IN THE SOFTWARE.
  */
 
-namespace test_harness {
+#ifndef SCOPED_TYPES_H
+#define SCOPED_TYPES_H
+
+extern "C" {
+#include "test_util.h"
+}
 
+namespace test_harness {
 class scoped_cursor {
     public:
     scoped_cursor() = default;
-    scoped_cursor(WT_SESSION *session, const char *uri, const char *cfg)
-    {
-        reinit(session, uri, cfg);
-    }
-
-    virtual ~scoped_cursor()
-    {
-        if (_cursor != nullptr)
-            testutil_check(_cursor->close(_cursor));
-    }
+    scoped_cursor(WT_SESSION *session, const char *uri, const char *cfg);
 
     /* Moving is ok but copying is not. */
-    scoped_cursor(scoped_cursor &&other)
-    {
-        std::swap(_cursor, other._cursor);
-    }
+    scoped_cursor(scoped_cursor &&other);
+
+    virtual ~scoped_cursor();
 
     /*
```

Comment:
This comment needs to go in the `cxx` since its describing how move assignment is implemented.

Reply:
Done.